etest/__init__.py

```diff
@@ -47,9 +47,10 @@ def echo_check_verbose(check):
 @click.option("-j", "--jobs", default=1, help="number of test to run simultaneously")
 @click.option("-q", "--quiet", is_flag=True, default=False, help="suppress all output")
 @click.option("-v", "--verbose", is_flag=True, default=False, help="provide more output")
+@click.option("-a", "--arch", default="amd64", help="architecture to test against")
 @click.version_option(information.VERSION)
 @click.argument("ebuilds", nargs=-1)
-def etest(dry_run, fast, jobs, quiet, verbose, ebuilds):
+def etest(dry_run, fast, jobs, quiet, verbose, arch, ebuilds):
```

Comment:
We should add type hints to this (at least the arch parameter).

Reply:
Is it really needed? Its provided by click. It would be easy to do tho.

Reply:
I argue that type hints should be everywhere.  I'll prep a diff that makes that more strict but I thought it was configured that way already.

Reply:
It isnt a bad idea, but considering that you left them out for the rest of parameters of this I thought they werent really necessary. I will look into adding them wherever possible on following commits.

Reply:
Ah, no, this code predates types in Python so part of updating and modernising the bits you're touching is adding those kinds of details.